charts/refarch-templates/values-example.yaml

```
@@ -160,6 +162,8 @@ refarch-gateway:
     # OpenShift-specific configuration, if you have plain Kubernetes, you will need to change the annotations, className.
     # annotations:
     #   route.openshift.io/termination: "edge"
+    # labels: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)